code/modules/food_and_drinks/food/snacks_pizza.dm

```diff
@@ -169,20 +169,25 @@
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 1, "pepperoni" = 2, "9 millimeter bullets" = 2)
 
 /obj/item/reagent_containers/food/snacks/proc/try_break_off(mob/living/M, mob/living/user) //maybe i give you a pizza maybe i break off your arm
```

Comment:
the fuck are these procs? looks like some LRP BS, consider removing it?

Reply:
wdym? arnold pizza?

Reply:
removing that proc would be feature removal which is not within the scope of this pr.

Reply:
then there is no need to update the pizza within this PR either if the feature is to be removed

Reply:
removing that proc would be feature removal which is not within the scope of this pr.